wof/examples/production_configs/wofpy.conf

```diff
@@ -15,5 +15,5 @@ setgid www-data
 env PATH=/home/ubuntu/miniconda/envs/wofpy/bin
 
 # change directory to where the uWSGI configuration file is located then execute uwsgi with the config file
-chdir /var/www/timeseries
-exec uwsgi --ini timeseries.ini
\ No newline at end of file
+chdir /var/www/wofpy
+exec uwsgi --ini wofpy.ini
```

Comment:
Perfect! Sorry I took so long to review this, and had so many question/misunderstandings.

This is awesome. Merging now.